### DISCUSSION TITLE
New to GitHub

### DISCUSSION THREAD

**Discussion by crn-droid** [community/community#103192] [New to GitHub]
Hi! everyone CRN here, I am a techie OfCourse 🦃🤣.I would like to announce that now I am on GitHub, and I would like to be involved in the community cheers🥂.

**Comment by alreadydea** [comment]
Hello

**Comment by Anixton** [comment]
Hi there

**Comment by queenofcorgis** [comment]
👋 Welcome to the GitHub Community on Discussions, @crn-droid  Happy to have you here 🙂

To help keep our space clutter-free, we recommend introducing yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204) and checking out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776). This post will be closed and locked to keep our community nice and tidy 🧹.
